desc/integrals/bounce_integral.py

```diff
@@ -439,11 +442,14 @@ def compute_theta(eq, X=16, Y=32, rho=1.0, iota=None, clebsch=None, **kwargs):
         iota : float or jnp.ndarray
             Shape (num rho, ).
             Optional, rotational transform on the flux surfaces to compute on.
-        clebsch : jnp.ndarray
-            Shape (num rho * X * Y, 3).
-            Optional, precomputed Clebsch coordinate tensor-product grid (ρ, α, ζ).
-            ``FourierChebyshevSeries.nodes(X,Y,rho,domain=(0,2*jnp.pi))``.
-            If supplied ``rho`` is ignored.
+        params : dict[str,jnp.ndarray]
+            Parameters from the equilibrium, such as R_lmn, Z_lmn, i_l, p_l, etc
+            Defaults to ``eq.params_dict``.
+        profiles
+            Optional profiles.
+        tol : float
+            Stopping tolerance for root finding.
+            Default is ``1e-7``.
         kwargs
             Additional parameters to supply to the coordinate mapping function.
             See ``desc.equilibrium.Equilibrium.map_coordinates``.
```

Comment:
`lmbda` is not in the kwargs of map_coordinates, should be mentioned here to avoid undocumented kwargs

Reply:
Got it, did not realize it was meant to be private